cartographer/sensor/point_cloud.h

```diff
@@ -27,7 +27,14 @@
 namespace cartographer {
 namespace sensor {
 
-typedef std::vector<Eigen::Vector3f> PointCloud;
+// Stores 3D positions of points with their measurement time
+// in the fourth entry.
+// Measurement time should be increasing and relative to the most
+// current measurement. The most current time entry is 0.f.
+// If timing is not available, all fourth entries are 0.f.
+// For 2D points, the third entry is 0.f and the fourth entry is time.
+// TODO(gaschler): Define a type PositionTime?
+typedef std::vector<Eigen::Vector4f> PointCloud;
 
 struct PointCloudWithIntensities {
```

Comment:
Remove 'offset_seconds' and make `points` a TimedPointCLoud

Reply:
Done